### PR TITLE
100th PR Bonanza: Corn kernels and cornucopiae for all!

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

File: src/code_F5750.c [hunk 1]
```diff
@@ -238,7 +238,9 @@ INCLUDE_ASM(s32, "code_F5750", func_802D2484);
 
 INCLUDE_ASM(s32, "code_F5750", func_802D249C);
 
-INCLUDE_ASM(s32, "code_F5750", func_802D24F4);
+ApiStatus func_802D24F4(ScriptInstance* script, s32 isInitialCall) {
+    return (gPlayerStatus.moveFrames == 0) * ApiStatus_DONE2;
```
Comment (nanaian) [inline src/code_F5750.c:242]: Sigh. Clover thought that they added DONE1 specifically to allow returning a bool, but I guess that's not the case. Maybe we'll find out what DONE1 does soon!

File: src/code_f2470_len_27f0.c [hunk 2]
```diff
@@ -35,11 +35,35 @@ ApiStatus GetNpcPointer(ScriptInstance* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
+#ifdef NON_MATCHING
+ApiStatus SetNpcPos(ScriptInstance* script, s32 isInitialCall) {
```
Comment (nanaian) [inline src/code_f2470_len_27f0.c:39]: What's non-matching about this? Would be good to add a comment.

File: src/code_f2470_len_27f0.c [hunk 3]
```diff
@@ -35,11 +35,35 @@ ApiStatus GetNpcPointer(ScriptInstance* script, s32 isInitialCall) {
...
+    if (npc == NULL) {
+        return ApiStatus_DONE2;
+    }
+
+    npc->pos.x = x;
+    npc->pos.y = y;
+    npc->pos.z = z;
+    npc->colliderPos.x = x;
+    npc->colliderPos.y = y;
+    npc->colliderPos.z = z;
```
Comment (nanaian) [inline src/code_f2470_len_27f0.c:56]: Can maybe do `npc->pos = { x, y, z }` if that's cleaner (I think so) for these if it matches.

File: src/world/area_mac/mac_00/7EAD10.c [hunk 4]
```diff
@@ -4,7 +4,33 @@
...
+        stickX = -stickX;
+    }
+
+    if ((stickX != 0) || (stickY != 0)) {
+        if (atan2(0.0f, 0.0f, stickX, stickY) < 60.0f) {
+            script->varTable[0] = 1;
+            return ApiStatus_DONE2;
+        }
+    }
+    return ApiStatus_BLOCK;
```
Comment (nanaian) [inline src/world/area_mac/mac_00/7EAD10.c:29]: Maybe a newline before this

**Review by nanaian — APPROVED** [review]
Looks good

File: src/code_f2470_len_27f0.c (hunk 3 above)
Comment (ethteck) [reply src/code_f2470_len_27f0.c:56]: I've never done something like this before. Is `{ x, y, z } a struct literal? If so, this is going to generate a struct copy which should be different codegen..but maybe that's why this doesn't match. I'll give it and other things a shot..

Comment (nanaian) [reply src/code_f2470_len_27f0.c:56]: Yep, it's a struct literal.

**Comment by nanaian** [comment]
Needs SetNpcPos.s deleted and ./format.sh ran, but other than LGTM

File: src/code_F5750.c (hunk 1 above)
Comment (pixel-stuck) [reply src/code_F5750.c:242]: I don't see how this negates done1 from being a bool. In this case, they're deciding if they should block or not based *on* a bool value, but aren't attempting to return a bool in particular, so I think that's still on the table?

**Review by pixel-stuck — APPROVED** [review]

**Pull request merged**